index.html

```diff
@@ -3,18 +3,15 @@
 <head>
   <link rel="stylesheet" href="styles.css">
   <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=<device-width>, initial-scale=1.0">
-  <link rel="preconnect" href="https://fonts.googleapis.com">
-  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  
   <title>Luis's Portfolio</title>
 </head>
 <body>
   <header>
-    <div id="welcome">Welcome</div>
+    <div id="welcome"><a>Welcome</a></div>
   <nav id="nav"><a href="#">
-      <img src="hamburger.png"></a>
+      <img src="hamburguer-button.png"></a>
     </nav>
   </header>
```

Comment:
- [ ] [OPTIONAL] Well done implementing all the changes requested before, however, I've just noticed some differences between your work and the Figma design that weren't addressed previously 🔽 I will categorize these changes as optional because they were missed before but I highly recommend you to implement them as they will improve your design quality, I apologize for missing these issues before 🙏🏽 

## Your Design
![2023-03-25_032109](https://user-images.githubusercontent.com/104319462/227689657-2cce4abc-d941-4eb8-ab3b-f229ad5feb30.png)

## Figma Design
![2023-03-25_032406](https://user-images.githubusercontent.com/104319462/227689654-ac6e876a-1864-464f-98d5-93cdb31ed7ce.png)